<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>view_ident_dialog</class>
 <widget class="QDialog" name="view_ident_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>119</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="viewIdentGroupbox">
     <property name="title">
      <string>View Identity: Identname</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="hashLabel">
          <property name="text">
           <string>ID hash: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="btcaddrLabel">
          <property name="text">
           <string>BTC address: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="bmaddrLabel">
          <property name="text">
           <string>BM address: </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
